spec/common/importers/lastpassCsvImporter.spec.ts

```diff
@@ -1,10 +1,29 @@
 import { LastPassCsvImporter as Importer } from '../../../src/importers/lastpassCsvImporter';
 
+import { ImportResult } from '../../../src/models/domain/importResult';
 import { CipherView } from '../../../src/models/view/cipherView';
 import { FieldView } from '../../../src/models/view/fieldView';
 
+import { CipherType } from '../../../src/enums';
 import { FieldType } from '../../../src/enums';
```

Comment:
Perhaps you can consolidate the imports from `../../../src/enums` as:
```typescript
import { CipherType, FieldType } from '../../../src/enums';
```

Reply:
Thank you @cscharf for the review.  I've fixed the import as requested.